README.md

```diff
@@ -157,6 +157,106 @@ This is useful for long examples.
 
 [^back to top](#recog-a-recognition-framework)
 
+### Examples
+
+The following examples make use of `bin/recog_match`, a simple Ruby command line tool that uses Recog's fingerprinting data. Pre-processing is generally required before running Recog, i.e. extracting HTTP header values, etc. All fingerprint data can be found in `xml/*.xml`.
```

Comment:
Consider replacing "that uses Recog's fingerprinting data" with "that uses Recog's fingerprint data".